test/fixtures/e2e/amp-story-player/pre-rendering.amp.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html>
```

Comment:
qq, do these still need the amp indicator?

Reply:
Yep it does I think, good catch! Will add it just in case.